src/main/kotlin/com/pubnub/api/endpoints/objects/internal/ReturningCustom.kt

```diff
@@ -77,8 +94,17 @@ data class ReturningChannelDetailsCustom @Deprecated("use Include", ReplaceWith(
     }
 }
 
-@Deprecated("use Include", ReplaceWith("Include"))
-data class ReturningUUIDDetailsCustom @Deprecated("use Include", ReplaceWith("Include(includeCustom = includeCustom, includeUUIDDetails = includeUUIDDetails)")) constructor(
+@Deprecated(
+    "use IncludeQueryParam",
+    ReplaceWith("IncludeQueryParam", "com.pubnub.api.endpoints.objects.internal.IncludeQueryParam")
+)
+data class ReturningUUIDDetailsCustom @Deprecated(
+    "use IncludeParam",
```

Comment:
seems that should be "use IncludeQueryParam" instead of "use IncludeParam"

Reply:
sure